packages/members-api/lib/stripe/index.js

```diff
@@ -128,9 +128,10 @@ module.exports = class StripePaymentProcessor {
             return subscription.status !== 'canceled';
         });
 
-        await Promise.all(activeSubscriptions.map((subscription) => {
-            return del(this._stripe, 'subscriptions', subscription.id);
-        }));
+        for (const subscription of activeSubscriptions) {
+            const updatedSubscription = await del(this._stripe, 'subscriptions', subscription.id);
+            await this._updateSubscription(updatedSubscription);
```

Comment:
We have to save subscription deletions locally right away to be able to send correct data back to the client (otherwise they are still returned from Admin API)

Reply:
:+1: Kinda sad we can't rely on webhooks here